### ISSUE TITLE
Can't login because my yubikey is not registered

### ISSUE THREAD

**Issue by stianlagstad** [ytmdesktop/ytmdesktop#110]
I installed with `sudo snap install youtube-music-desktop-app`, and I got v1.8.2. After entering my email and password, I'm prompted to enter my security key. I enter my yubikey but nothing happens. The page just keeps spinning. Is there a way I can use this app as long as my account is secured with yubikey?

![Screenshot from 2020-03-06 10-54-47](https://user-images.githubusercontent.com/4340859/76073213-8ab56480-5f99-11ea-90e2-26881c491e0c.jpg)

**Comment by stianlagstad** [comment]
This is probably similar to https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/3573.

**Comment by rahmnathan** [comment]
Not sure if it's useful, but this is the only other snap I use that supports YubiKeys - https://github.com/keepassxreboot/keepassxc/issues/1342. Maybe their implementation could be used as a reference.

**Comment by rickinbsb** [comment]
Someone suggested on the GPMDP issue that upgrading to electron 4 would fix this, but since we are on electron 7 I believe there is something else causing this. I believe some debugging is necessary here, but we don't have yubikeys. Help on investigating this would be highly appreciated

**Comment by rickinbsb** [comment]
can you check if yubikey works when installing this via .appimage?

**Comment by rahmnathan** [comment]
@rickinbsb Absolutely. Can you point me to the appimage? I can't seem to find them.

**Comment by rickinbsb** [comment]
Here: [YouTube.Music.Desktop.App-1.9.0.AppImage](https://github.com/ytmdesktop/ytmdesktop/releases/download/v1.9.0/YouTube.Music.Desktop.App-1.9.0.AppImage). It's not yet updated to 1.10 but I'll address this later today

**Comment by rahmnathan** [comment]
@rickinbsb Confirmed that U2F/Yubikey works with the AppImage.

**Comment by rickinbsb** [comment]
Thanks! I'll check how KeePassXC fixed this

**Comment by rahmnathan** [comment]
@rickinbsb I was looking into how to add this, but can't seem to find snapcraft configuration in this project. Do you know how the snaps are being generated?

**Comment by rickinbsb** [comment]
they are generated with electron-builder. config is on electron-builder64.yml

**Comment by jippylong12** [comment]
This issue may be obsolete for v2. The app no longer ships as a snap (v2 uses Electron Forge with deb/rpm/zip). The original bug was a missing u2f-devices plug in the snap sandbox config. If snap support isn't planned for v2, recommend closing. @Alipoodle

**Comment by tjkirch** [comment]
> This issue may be obsolete for v2. The app no longer ships as a snap (v2 uses Electron Forge with deb/rpm/zip). The original bug was a missing u2f-devices plug in the snap sandbox config. If snap support isn't planned for v2, recommend closing. [@Alipoodle](https://github.com/Alipoodle)

It affected more than just snap - the title may be a bit too specific.
